packages/jira-adapter/src/filters/account_info.ts

```diff
@@ -38,69 +75,77 @@ const LICENSE_RESPONSE_SCHEME = Joi.object({
+    },
+    undefined,
+    { [CORE_ANNOTATIONS.HIDDEN]: true },
+  )
+  return accountInfoInstance
+}
+
 /*
  * Brings account info and stores it in an hidden nacl
  * the info includes license data
 */
 const filter: FilterCreator = ({ client }) => ({
   onFetch: async elements => log.time(async () => {
-    if (client.isDataCenter) {
-      // Add here data center license info in the future
-      return
-    }
-    const licensedApplications = new ObjectType({
-      elemID: new ElemID(JIRA, LICENSED_APPLICATION_TYPE),
-      isSettings: true,
-      fields: {
-        id: { refType: BuiltinTypes.STRING },
-        plan: { refType: BuiltinTypes.STRING },
-      },
-      annotations: {
-        [CORE_ANNOTATIONS.HIDDEN]: true,
-      },
-    })
-    const licenseType = new ObjectType({
-      elemID: new ElemID(JIRA, LICENSE_TYPE),
-      isSettings: true,
-      fields: {
-        applications: { refType: new ListType(licensedApplications) },
-      },
-      annotations: {
-        [CORE_ANNOTATIONS.HIDDEN]: true,
-      },
-    })
-    const accountInfoType = new ObjectType({
-      elemID: new ElemID(JIRA, ACCOUNT_INFO_TYPE),
-      isSettings: true,
-      fields: {
-        license: { refType: new ListType(licenseType) },
-      },
-      annotations: {
-        [CORE_ANNOTATIONS.HIDDEN]: true,
-      },
-    })
+    const [accountInfoType, licenseType, licensedApplications] = createAccountTypes()
     try {
-      const response = await client.getSinglePage({
-        url: '/rest/api/3/instance/license',
-      })
-      if (!isLicenseResponse(response.data)) {
-        throw new Error('Received an invalid license response')
-      }
-      const accountInfoInstance = new InstanceElement(
-        ElemID.CONFIG_NAME,
-        accountInfoType,
-        {
-          license: {
-            applications: response.data.applications,
-          },
-        },
-        undefined,
-        { [CORE_ANNOTATIONS.HIDDEN]: true },
-      )
-      elements.push(licenseType, licensedApplications, accountInfoType, accountInfoInstance)
+      elements.push(licenseType, licensedApplications, accountInfoType, await getAccountInfo(client, accountInfoType))
     } catch (e) {
-      log.error(`Received an error when fetching jira license, ${e.message}.`)
+      if (e instanceof clientUtils.HTTPError && e.message !== undefined) {
+        log.error(`Received an error when fetching jira license, ${e.message}.`)
+      } else {
+        log.error('Received a non http error when fetching jira license.', e)
+      }
     }
+
+    // take user count from application roles and delete it from the nacl
+    const applicationRoles = elements
+      .filter(element => element.elemID.typeName === 'ApplicationRole')
+      .filter(isInstanceElement)
+    log.info('jira user count is: %s',
+      applicationRoles
+        .filter(role => role.value.key === 'jira-software')
+        .map(role => role.value.userCount)[0] ?? 'unknown')
```

Comment:
nit: if you only need the first one its better to use find

```suggestion
    log.info('jira user count is: %s',
      applicationRoles
        .find(role => role.value.key === 'jira-software')?.value.userCount ?? 'unknown')
```